actions/bubble_merge/git_client.rb

```diff
@@ -78,6 +81,26 @@ def pr_number
       event.dig('issue', 'number')
     end
 
+    def pr_status
+      checks = head_commit_checks[:check_suites]
+
+      @pr_status = if checks.any? { |check| check[:status] != 'completed' }
```

Comment:
Why do we assign this to a class variable? Should this be `@pr_status ||= ...`? (This may also be holiday brain)

Reply:
I totally forgot to remove this. Thanks!